packages/wormhole-sdk/src/domains.ts

```diff
@@ -38,6 +40,25 @@ export function getDefaultDstDomain(srcDomain: DomainId): DomainId {
   throw new Error(`No default destination domain for source domain "${srcDomain}"`)
 }
 
+export function getLikelyDomainId(description: DomainDescription): DomainId {
+  switch (description) {
+    case 'RINKEBY-SLAVE-ARBITRUM-1':
+    case 'arbitrum-testnet':
+      return 'RINKEBY-SLAVE-ARBITRUM-1'
+    case 'KOVAN-SLAVE-OPTIMISM-1':
+    case 'optimism-testnet':
+      return 'KOVAN-SLAVE-OPTIMISM-1'
+    // case 'ETHEREUM-SLAVE-OPTIMISM-1':
+    // case 'optimism':
+    //   return 'ETHEREUM-SLAVE-OPTIMISM-1'
+    // case 'ETHEREUM-SLAVE-ARBITRUM-1':
+    // case 'arbitrum':
+    //   return 'ETHEREUM-SLAVE-ARBITRUM-1'
+    default:
```

Comment:
I think you should use something like: https://github.com/ts-essentials/ts-essentials#Exhaustive-switch-cases

to ensure that we are not missing any cases, or use a dictionary instead.

Reply:
I didn't include the destination domains here, because they do not have corresponding domain descriptions (not needed as the destination domain can currently be guessed based on the source domain). I'll add them for completeness and future-proofing.

Reply:
I got what you meant re: missing cases. Will use ts-essentials.